tests/platforms/web/static/conftest.py

```diff
@@ -16,27 +16,39 @@ def first_app_generated(first_app_config, tmp_path):
 """,
     )
 
-    # Create index.html
-    create_file(bundle_path / "www/index.html", "<html></html>")
-
-    # Create the initial pyscript.toml
+    # Create index.html with insert markers
     create_file(
-        bundle_path / "www/pyscript.toml",
-        """
-existing-key-1 = "value-1"
-existing-key-2 = 2
+        bundle_path / "www/index.html",
+        """<!doctype html>
+<html>
+  <head>
+    <!--@@ head:start @@-->
+    <!--@@ head:end @@-->
+    <!--@@ Python:start @@-->
+    <!--@@ Python:end @@-->
```

Comment:
Minor consistency note - this should probably be `python`, so that we can predict insert title case.